.github/workflows/docker-push.yaml

```diff
@@ -11,22 +11,25 @@ jobs:
     env:
       DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
       DOCKER_USER: ${{ secrets.DOCKER_LOGIN }}
-      TAG: ${GITHUB_SHA::8}
       ORG: networkservicemeshci
       CGO_ENABLED: 0
       NAME: ${{ github.event.repository.name }}
     if: github.repository != 'networkservicemesh/cmd-template'
     steps:
       - uses: actions/checkout@v2
+      - uses: benjlevesque/short-sha@v1.1
```

Comment:
Question... why this instead of just the simpler: ${GITHUB_SHA:0:8}

Reply:
Yes, I guess it is a more clear variant. Fix: https://github.com/networkservicemesh/cmd-template/pull/32